src/Template/Layout/js/app/components/filter-box.js

```diff
@@ -21,9 +21,9 @@ import { DEFAULT_PAGINATION } from 'app/mixins/paginated-content';
 export default {
     template:
     `
-        <nav class="pagination has-text-size-smallest" :class="pagination.count > 4 && 'show-pagination'">
+        <nav class="pagination has-text-size-smallest" :class="pagination.count > 10 && 'show-pagination'">
```

Comment:
instead of `10` should this value be the minimum or first from `{{ config('Pagination.sizeAvailable') }}` ?